_rules/meta-refresh-no-delay-bc659a.md

```diff
@@ -54,7 +54,7 @@ This rule applies to the first `meta` element in a document for which all the fo
 
 ## Expectation
 
-For each test target, running the [shared declarative refresh steps][], given the target's document, the value of the target's `content` attribute, and the target results in _time_ being either 0 or greater than 72000 (20 hours).
+For each test target, running the [shared declarative refresh steps][] results in _time_ being 0 or greater than 72000 (20 hours).
```

Comment:
I don't thing https://github.com/act-rules/act-rules.github.io/pull/1831#discussion_r860798971 has been resolved.
The shared declarative refresh steps expect some inputs which I do think are ambiguous here (the document could be the target's document or the target's top-level browsing context document, for example). 

It makes the new expectation ambiguous.

Reply:
DO you have a suggestion?

Reply:
Not really… The previous formulation was saying was are the arguments to the algorithm. These changes are requested by TF, I am not part of it so I do not know the discussions and the reasons why TF wants that to be changed…
As far as I am concerned, the previous formulation was OK 🤷

Reply:
I made this suggestion, since I thought that the previous expectation was just duplicating information that was already available inside of the input aspects and the applicability (which I still kind of believe). I didn't think about the target's document vs the target's top-level browsing context, though I'm not sure this would impact the rule significantly in practice. And from your comment, I do see that the previous definition was using the same format given by the `shared declarative refresh steps` algorithm. At this point I could go either way on it.